geode-core/src/main/java/org/apache/geode/internal/net/NioPlainEngine.java

```diff
@@ -17,24 +17,31 @@
 
 import java.io.EOFException;
```

Comment:
should these classes be renamed since we are using OiO now?

Reply:
Those classes are no longer used by the tcp package, but will be used by client/server.  I don't think they should be renamed.